packages/squidex/schema/schemas/research-outputs.json

```diff
@@ -213,6 +213,186 @@
+            "3D Printing",
+            "Animal Models",
+            "Antibodies",
+            "Biosample",
+            "Cell line",
+            "Compounds",
+            "Plasmid",
+            "Protein",
+            "Viral Vector"
+          ],
+          "contentType": "Unspecified",
+          "label": "Subtype"
+        },
+        "isLocked": false,
+        "isHidden": false,
+        "isDisabled": false,
+        "partitioning": "invariant"
+      },
+      {
+        "name": "accessInstructions",
+        "properties": {
+          "isRequired": false,
+          "isRequiredOnPublish": false,
+          "isHalfWidth": false,
+          "fieldType": "String",
+          "editor": "Input",
+          "inlineEditable": false,
+          "isUnique": false,
+          "contentType": "Unspecified",
+          "label": "Access Instructions"
+        },
+        "isLocked": false,
+        "isHidden": false,
+        "isDisabled": false,
+        "partitioning": "invariant"
+      },
+      {
+        "name": "sharingStatus",
+        "properties": {
+          "isRequired": true,
+          "isRequiredOnPublish": false,
+          "isHalfWidth": false,
+          "fieldType": "String",
+          "editor": "Dropdown",
+          "inlineEditable": false,
+          "isUnique": false,
+          "defaultValue": "Network Only",
+          "allowedValues": ["Public", "Network Only"],
+          "contentType": "Unspecified",
+          "label": "Sharing Status"
+        },
+        "isLocked": false,
+        "isHidden": false,
+        "isDisabled": false,
+        "partitioning": "invariant"
+      },
+      {
+        "name": "asapFunded",
+        "properties": {
+          "isRequired": true,
+          "isRequiredOnPublish": false,
+          "isHalfWidth": false,
+          "fieldType": "String",
+          "editor": "Dropdown",
+          "inlineEditable": false,
+          "isUnique": false,
+          "defaultValue": "Not Sure",
+          "allowedValues": ["Yes", "No", "Not Sure"],
+          "contentType": "Unspecified",
+          "label": "ASAP Funded"
+        },
+        "isLocked": false,
+        "isHidden": false,
+        "isDisabled": false,
+        "partitioning": "invariant"
+      },
+      {
+        "name": "usedInAPublication",
+        "properties": {
+          "isRequired": true,
+          "isRequiredOnPublish": false,
+          "isHalfWidth": false,
+          "fieldType": "String",
+          "editor": "Dropdown",
+          "inlineEditable": false,
+          "isUnique": false,
+          "defaultValue": "Not Sure",
+          "allowedValues": ["Yes", "No", "Not Sure"],
+          "contentType": "Unspecified",
+          "label": "Used in a Publication"
+        },
+        "isLocked": false,
+        "isHidden": false,
+        "isDisabled": false,
+        "partitioning": "invariant"
+      },
+      {
+        "name": "externalAuthors",
+        "properties": {
+          "isRequired": false,
```

Comment:
should we not make this required so its always an empty array and not an optional property? it would make it easier to type and use but it may not be in line with our previous approach to nested fields

Reply:
Since this is a field that could get entries manually deleted, this one could easily end up in the `null` state we witnessed on the `avatar`. This way we will write safer code, what do you think? Either way, we can always send it as an array for the FE